### ISSUE TITLE
comment on files in a pull request in NostrGit?

### ISSUE THREAD

**Issue by rusteroobs** [NostrGit/NostrGit#223]
Hello,

Recently, GitHub added a comment feature on pull-request, see this: [2023-03-14-comment-on-files-in-a-pull-request-public-beta](https://github.blog/changelog/2023-03-14-comment-on-files-in-a-pull-request-public-beta/). My question is whether there is interest in adding a comment to the pull-request on NostrGit.

**Comment by AntwanSherif** [comment]
For a long time I wanted such a feature where you can comment on why you have added/deleted a file.
I think this would be a nice feature.

**Comment by cypherhoodlum** [comment]
Yes, sure. I think nostr makes this very trivial to implement once we have implemented pull-requests as nostr events in the first place.

**Comment by marcelolucario** [comment]
Hi cypherhoodlum and AntwanSherif , thanks for feedback.
